docs/installation/install.md

```diff
@@ -120,6 +120,8 @@ Alternatively, you can manually set a different name with the `--cluster-name` *
 ```{admonition} Note
 If you are running an [AKS private cluster](https://learn.microsoft.com/en-us/azure/aks/private-clusters), you may need to set the `--disable-api-server-sanity-check` *liqoctl* flag, since the API Server in your kubeconfig may be different from the one retrieved from the Azure APIs.
 
+If the private cluster uses private link, you can set the `--private-link` *liqoctl* flag to use the private FQDN for the API server.
```

Comment:
@scottboring Why are we defining a flag named 'private-link' to set a DNS name?
To me, it looks more intuitive something as follows:

If the private clusters use a private link, you can use set the `--private-api-server` *liqoctl* flag to use the private FQDN for the API server.

What do you think?

Reply:
Hi @frisso, this flag is related to the AKS provider and is used to tell liqoctl to get the API Server address not from the public but from the private one (called `private-link` in Azure).

We already have another flag to set a generic different name for the API Server.

Reply:
Got the point. However, we have to make sure that who reviews the documentation, even if it is not 100% expert of Azure, feels comfortable with what he's reading. The current sentence looks strange to me. What about:

If the private cluster uses private link, you can set the `--private-link` *liqoctl* flag, which configures this tool to use the private FQDN for the API server instead of the default (public) one.

Reply:
Perfectly fine for me